docs/guide.md

```diff
@@ -56,7 +56,11 @@ webdriverio
     .end();
 ```
 
-** 6. Run your test file**
+** 6. Install browser and appropriate driver**
+
+In the example above FireFox is specified as the desired browser, but may not be currently installed on the testing system. The choice is to either install FireFox or download [Third Party Drivers](http://docs.seleniumhq.org/download/#thirdPartyDrivers) and place them in the same directory as the selenium standalone server.
```

Comment:
> and place them in the same directory as the selenium standalone server.

What does this mean? Just put the binary in the same directory as your jar file? I don't thin that is how it works